client/state/global-sidebar/selectors.ts

```diff
@@ -20,5 +20,9 @@ export const getShouldShowGlobalSiteSidebar = (
 	sectionGroup: string
 ) => {
 	// Global Site View should be limited to classic interface users only for now.
-	return isGlobalSiteViewEnabled( state, siteId ) && sectionGroup === 'sites' && !! siteId;
+	return (
+		isGlobalSiteViewEnabled( state, siteId ) &&
+		( sectionGroup === 'sites' || sectionGroup === 'jetpack-cloud' ) &&
+		!! siteId
+	);
```

Comment:
Am I right to assume we don't want global side bar shown on jetpack-cloud routes?

This looks like the route provider is giving us bad inputs.

Is it possible to fix that instead?

If not maybe just leave a comment explaining why jetpack-cloud isn't actually jetpack-cloud anywhere isGlobalSiteViewEnabled is returning true.

Reply:
`section.js` now has two ([1](https://github.com/Automattic/wp-calypso/blob/trunk/client/sections.js#L208), [2](https://github.com/Automattic/wp-calypso/blob/trunk/client/sections.js#L659)) definitions for the `/subscribers` path. The latter was introduced in the recent PR https://github.com/Automattic/wp-calypso/pull/87684, and since it's defined later in the sections array, it always takes precedence.

Another approach is to add a middle to the subscribers controller to update the `currentSection`, like implemented in this PR https://github.com/Automattic/wp-calypso/pull/88542.

Either approach works for me.